lib/feature-flags/feature-flags.tsx

```diff
@@ -65,3 +67,12 @@ export const ConditionalFeature: React.FC<{ name: string }> = ({
 
   return <>{children}</>;
 };
+
+/**
+ * Typescript helper function for checking whether a given feature is active or inactive
+ */
+export const isFeatureFlagActive = (featureName: string) => {
+  const features = getFeatureFlags();
+
+  return features[featureName].isActive;
```

Comment:
I think we can reuse the `isFeatureActive` function here, which will mean we also check that the feature is on the object before looking for the `isActive` attribute. Something like:

```tsx
return isFeatureActive(features)(featureName);
```